src/test/java/codeu/model/store/basic/ConversationStoreTest.java

```diff
@@ -52,6 +55,12 @@ public void testIsTitleTaken_true() {
     Assert.assertTrue(isTitleTaken);
   }
 
+  @Test
+  public void testGetSize() {
+    int conversationSize = mockConversationStore.getSize();
```

Comment:
What does having the conversationSize variable do (along with the other _____Size variables in the respective tests)? I feel like the Mockito.verify().getSize() takes care of making sure the call work, and since the variable isn't called in the rest of the function, you don't need to create it. BUT IF YOU DO THEN IDK SORRY LOL

Also maybe you can do checks for null/non-null conversation/user/message lists to make sure everything works when lists are null.

Reply:
Whoops, you're correct. I forgot to remove that line from all three